pkgs/development/libraries/grilo/default.nix

```diff
@@ -34,6 +34,11 @@ stdenv.mkDerivation rec {
     sha256 = "81Ks9zZlZpk0JwY2/t5mtS2mgB/iD2OMQEirJnhXey0=";
   };
 
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    substituteInPlace src/meson.build \
```

Comment:
Could you please upstream this, see e.g. https://gitlab.gnome.org/GNOME/babl/-/merge_requests/26

Reply:
https://gitlab.gnome.org/GNOME/grilo/-/merge_requests/96

Reply:
Upstreamed.